### ISSUE TITLE
spawned in enemies warp away asap

### ISSUE THREAD

**Issue by commche** [tsunamayo/Starship-EVO#4785]
### Short Description of the issue:

Spawned in enemies warp away. My CPU isn't the best so if I spawn in ships with lots of CE, by the time my game is unfrozen, the ship has warped away. basically enemy AI is AS (artificial stupidity)

### Steps to reproduce the issue:

Ryzen 3 CPU. Spawn in a complex ship for battle. Wait for 3 mins to unfreeze PC. Ship has gone.

### Environment
- Starship EVO version: latest
- Graphics Card: radeon armor 570

I think enemy ship AI needs to be more aggressive, and not have an escape imperative

**Comment by tsunamayo** [comment]
@commche Hi, are your sure the spawned ship has weapons? Can you send me the blueprint?
Thanks

**Comment by commche** [comment]
Yeah it was Dolan's Zeke Class Corvette. I think it was more of a performance issue on my end, because the game froze for about 2 minutes before anything happened - because of the hundreds of entities between our two ships. Im running a Ryzen 3, which is just not fast enough for big battles. Otherwise the game performs amazing on my rig. The "escape" bug only happens when I get slowdown or as you said, if one party has no weapons.

**Comment by Vincent14** [comment]
I've also seen ships warping after be spawned from F8 menu
